### ISSUE TITLE
Add default filters for real-time events if service is generated with authentication required

### ISSUE THREAD

**Issue by MichaelErmer** [feathersjs/feathers-cli#31]
Currently if you create a service using feathers generate service with 

    Does your service require users to be authenticated? Yes

the following hooks are applied:
```
all: [
    auth.verifyToken(),
    auth.populateUser(),
    auth.restrictToAuthenticated()
  ]
```

The problem is there are no default filters for live events, which exposes a huge security threat for people using generator and not aware of this.

People can just connect to the server unauthorized and catch all created/update/etc. events:

    feathers.service('messages').on('updated', function(obj) { console.log(obj); })

The generator should at least add a simple filter to the service like the one from the docs http://docs.feathersjs.com/real-time/filtering.html#filter-examples

**Comment by marshallswain** [comment]
This is a good point, and one that we've notice before.  We are planning on not sending events by default in the next release of Feathers.

**Comment by MichaelErmer** [comment]
Ok, great. Might be worthy to add a simple filter to the service generator until the next release.

    service.filter(function(data, connection) { return connection.user ? data : false; });

**Comment by daffl** [comment]
This will be added in the new CLI. See https://github.com/feathersjs/feathers-bootstrap/blob/master/example/src/services/messages/messages.filter.js

**Comment by slajax** [comment]
closed by feathersjs/feathers-generator#10
